packages/airnode-protocol/example.env

```diff
@@ -0,0 +1,24 @@
+MNEMONIC=""
+ETHERSCAN_API_KEY_arbitrum-goerli-testnet=""
+ETHERSCAN_API_KEY_arbitrum-nova=""
+ETHERSCAN_API_KEY_arbitrum=""
+ETHERSCAN_API_KEY_avalanche-testnet=""
+ETHERSCAN_API_KEY_avalanche=""
+ETHERSCAN_API_KEY_boba-ethereum=""
```

Comment:
These should match the available chains in @api3/chains.CHAINS, right? We keep boba and sx deployments for reference but new deployments will only care about supported chains listed in @api3/chains

Reply:
I mean @api3/chains to support all chains that we do anything on https://github.com/api3dao/chains/issues/2